### PR TITLE
[300] Created unit test to CustomShoppingListResponseDtoMapper

### PR DESCRIPTION
**Pull request by Maryna-511750**

### CODE REVIEW

File: service/src/test/java/greencity/mapping/CustomShoppingListResponseDtoMapperTest.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(SpringExtension.class)
+class CustomShoppingListResponseDtoMapperTest {
+    private final ModelMapper modelMapper = new ModelMapper();
+    private final CustomShoppingListResponseDtoMapper customShoppingListResponseDtoMapper = new CustomShoppingListResponseDtoMapper();
+    @Test
```
Comment (OlehTsipotan) [inline service/src/test/java/greencity/mapping/CustomShoppingListResponseDtoMapperTest.java]: add empty lines

File: service/src/test/java/greencity/mapping/CustomShoppingListResponseDtoMapperTest.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+import greencity.enums.ShoppingListItemStatus;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.modelmapper.ModelMapper;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (OlehTsipotan) [inline service/src/test/java/greencity/mapping/CustomShoppingListResponseDtoMapperTest.java]: fix import order

File: service/src/test/java/greencity/mapping/CustomShoppingListResponseDtoMapperTest.java [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        assertNotNull(actual);
+        assertEquals(2, actual.size());
+    }
+
+    @Test
+    void mapAllToList_CustomShoppingListResponseDtoMapperTest_NullPointerException() {
+        assertThrows(NullPointerException.class, () -> {
+            customShoppingListResponseDtoMapper.mapAllToList(null);
+        });
+    }
```
Comment (OlehTsipotan) [inline service/src/test/java/greencity/mapping/CustomShoppingListResponseDtoMapperTest.java]:
```suggestion
    @ParametrizedTest
    @NullSource
    void mapAllToList_whenCustomShoppingListItemListIsNull_throwNullPointerException(List<CustomShoppingListItem> nullList) {
        assertThrows(NullPointerException.class, () -> {
            customShoppingListResponseDtoMapper.mapAllToList(nullList);
        });
    }
```

File: service/src/test/java/greencity/mapping/CustomShoppingListResponseDtoMapperTest.java [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+    @Test
+    void convert_CustomShoppingListResponseDtoMapperTest_ShouldReturnNullPointerException() {
+        CustomShoppingListItem customShoppingListItem = null;
+
+        assertThrows(NullPointerException.class, () -> {
+            customShoppingListResponseDtoMapper.convert(customShoppingListItem);
+        });
+    }
```
Comment (OlehTsipotan) [inline service/src/test/java/greencity/mapping/CustomShoppingListResponseDtoMapperTest.java]: use `@NullSource`

File: service/src/test/java/greencity/mapping/CustomShoppingListResponseDtoMapperTest.java [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+    @Test
+    void mapAllToList_CustomShoppingListResponseDtoMapperTest_NullPointerException() {
+        assertThrows(NullPointerException.class, () -> {
+            customShoppingListResponseDtoMapper.mapAllToList(null);
+        });
+    }
+
+    @Test
+    void mapAllToList_CustomShoppingListResponseDtoMapperTest_ShouldReturnEmptyList() {
+        List<CustomShoppingListItem> dtoList  = Arrays.asList();
```
Comment (OlehTsipotan) [inline service/src/test/java/greencity/mapping/CustomShoppingListResponseDtoMapperTest.java]: I`m not sure but u can try - @EmptySource

**Review by OlehTsipotan — COMMENTED** [review]
Just a few spaces to improve

**Review by ryhaldenys — APPROVED** [review]

**Pull request merged**